team-app/src/Search.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+
+export default Search = ({ setFilteredCards, teammates }) => {
+  const [searchInput, setSearchInput] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    try {
+      event.preventDefault();
+      setErrorMessage("");
+
+      if (searchInput != "" && searchInput.length < 3) {
+        setErrorMessage("Please enter at least 3 letters");
+        return;
+      }
+
+      const filteredMembers = filterMembers();
+      filteredMembers.length > 0
+        ? setFilteredCards(filteredMembers)
+        : setErrorMessage("Member not found");
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const filterMembers = () => {
+    return teammates.filter((member) => {
```

Comment:
can be returned directly without return and curly braces

Reply:
my bad. will remove that.